test/common/orca/orca_parser_test.cc

```diff
@@ -83,7 +94,7 @@ TEST(OrcaParserUtilTest, NativeHttpEncodedHeader) {
 TEST(OrcaParserUtilTest, NativeHttpEncodedHeaderIncorrectFieldType) {
   Http::TestRequestHeaderMapImpl headers{
       {std::string(kEndpointLoadMetricsHeader),
-       absl::StrCat(kHeaderFormatPrefixText, "cpu_utilization:\"0.7\"")}};
+       absl::StrCat(kHeaderFormatPrefixText, "cpu_utilization=\"0.7\"")}};
```

Comment:
nit: let's leave one of the tests with ':', just to make sure our tests cover that path and future changes don't break this ability.

Reply:
@adisuissa See the `NativeHttpEncodedHeaderWithColon`. I added a test for that. But because it's same with the old code, so you may didn't notice it.

Reply:
Ack, missed it.
Thanks!